packages/engine/Source/Core/TrackingReferenceFrame.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Constants for identifying well-known tracking reference frames.
+ *
+ * @enum {number}
+ */
+const TrackingReferenceFrame = {
+  /**
+   * Auto-detect algorithm. The reference frame used to track the Entity will
+   * be automatically selected based on its trajectory: near-surface slow moving
+   * objects will be tracked in the entity's local east-north-up reference
+   * frame, while faster objects like satellites will use VVLH (Vehicle Velocity,
+   * Local Horizontal).
+   *
+   * @type {number}
+   * @constant
+   */
+  AUTODETECT: 0,
+
+  /**
+   * The entity's inertial reference frame. If entity has no defined orientation
+   * property, a {@link VelocityOrientationProperty} is used instead, thus
+   * falling back to TrackingReferenceFrame.VELOCITY.
+   * When selected, the auto-detect algorithm is overridden.
+   *
+   * @type {number}
+   * @constant
+   */
+  INERTIAL: 1,
```

Comment:
Would it make sense to fall back to autodetect in the case where an entity has no defined orientation?

Reply:
I agree on that point, as we already have `TrackingReferenceFrame.VELOCITY`. I'll change that!